app/controllers/daily_reports_controller.rb

```diff
@@ -0,0 +1,54 @@
+class DailyReportsController < ApplicationController
+  before_action :check_user_role, only: %i(new create)
+  # GET /daily_reports/new
+  def new
+    @daily_report = DailyReport.new
+  end
+
+  # POST /daily_reports
+  def create
+    @daily_report = DailyReport.new daily_report_params
+    @daily_report.user_id = current_user.id
```

Comment:
```suggestion
    @daily_report = DailyReport.new daily_report_params.merge(user_id: current_user.id)
```
Merge vào luôn cho gọn code